apple/native/NativeProxy.mm

```diff
@@ -254,7 +254,7 @@ - (void)dispatchViewManagerCommand:(nonnull NSNumber *)reactTag
   };
 
   auto nativeReanimatedModule = std::make_shared<NativeReanimatedModule>(
-      rnRuntime, jsInvoker, jsQueue, uiScheduler, platformDepMethodsHolder, valueUnpackerCode);
+      rnRuntime, jsInvoker, jsQueue, uiScheduler, platformDepMethodsHolder, valueUnpackerCode, nullptr);
```

Comment:
same think like with `JSScheduler` lets create two constructors

Reply:
Done in [2efade7](https://github.com/software-mansion/react-native-reanimated/pull/5734/commits/2efade7e68c277b2ab8ca61c31814a910dce2703)